packages/launch/src/focused-launch/summary/index.tsx

```diff
@@ -490,6 +491,7 @@ const Summary: React.FunctionComponent = () => {
 	const { onDomainSelect, onExistingSubdomainSelect, currentDomain } = useDomainSelection();
 	const { domainSearch, isLoading } = useDomainSearch();
 	const { isPaidPlan: hasPaidPlan } = useSite();
+	const hasPlanInCart = usePlanFromCart();
```

Comment:
Nitpick, but the `hasPlanInCart` variable name suggests a boolean value, while `usePlanFromCart()` returns a `Plan` object (or `undefined`).

```suggestion
	const planFromCart = usePlanFromCart();
```

Reply:
Not applcable after https://github.com/Automattic/wp-calypso/pull/48759/commits/b48ba4839033a39ed6006f81fcf633d25b8769b7